lms/static/js/discussions_management/views/divided_discussions.js

```diff
@@ -14,53 +15,54 @@
-                * @returns {Array} - Cohorted discussions.
+                * @returns {Array} - Divided discussions.
                 */
-                getCohortedDiscussions: function(selector) {
+                getDividedDiscussions: function(selector) {
                     var self = this,
-                        cohortedDiscussions = [];
+                        dividedDiscussions = [];
 
                     _.each(self.$(selector), function(topic) {
-                        cohortedDiscussions.push($(topic).data('id'));
+                        dividedDiscussions.push($(topic).data('id'));
                     });
-                    return cohortedDiscussions;
+                    return dividedDiscussions;
                 },
 
                 /**
-                * Save the cohortSettings' changed attributes to the server via PATCH method.
+                * Save the discussionSettings' changed attributes to the server via PATCH method.
                 * It shows the error message(s) if any.
                 * @param {object} $element - Messages would be shown before this element.
                 * @param {object} fieldData - Data to update on the server.
                 */
                 saveForm: function($element, fieldData) {
                     var self = this,
-                        cohortSettingsModel = this.cohortSettings,
+                        discussionSettingsModel = this.discussionSettings,
                         saveOperation = $.Deferred(),
                         showErrorMessage;
-
-                    showErrorMessage = function(message, $element) {
+                    showErrorMessage = function(message) {
                         self.showMessage(message, $element, 'error');
                     };
                     this.removeNotification();
 
-                    cohortSettingsModel.save(
+                    discussionSettingsModel.save(
                         fieldData, {patch: true, wait: true}
                     ).done(function() {
                         saveOperation.resolve();
                     }).fail(function(result) {
-                        var errorMessage = null;
+                        var errorMessage = null,
+                            jsonResponse;
                         try {
-                            var jsonResponse = JSON.parse(result.responseText);
+                            jsonResponse = JSON.parse(result.responseText);
                             errorMessage = jsonResponse.error;
                         } catch (e) {
                             // Ignore the exception and show the default error message instead.
                         }
                         if (!errorMessage) {
-                            errorMessage = gettext("We've encountered an error. Refresh your browser and then try again.");
+                            errorMessage = gettext('We\'ve encountered an error. ' +
```

Comment:
You can't break an argument to gettext in JS files-- see http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#do-not-append-strings-or-interpolate-values.

> Note that you cannot concatenate (+) within the gettext call at all